src/status_im/navigation/screens.cljs

```diff
@@ -966,7 +966,13 @@
    {:name      :screen/keycard.create.ready-to-add
     :metrics   {:track? true}
     :options   {:insets {:top? true :bottom? true}}
-    :component keycard.create/ready-to-add}])
+    :component keycard.create/ready-to-add}
+
+   {:name      :screen/keycard.enter-seed-phrase
```

Comment:
could you elaborate please on why new screen id is needed?

Reply:
Hi @flexsurfer,

Thanks so much for reviewing the PR.

I initially tried using `:screen/onboarding.enter-seed-phrase` for this flow, but that screen id has some hardcoded elements like `:layout options/onboarding-transparent-layout`, `:animations transitions/push-animations-for-transparent-background`, and is tailored specifically to the onboarding flow (e.g., with use dynamic overlay and navigate-to-within-stack).
It didn’t work well for the keycard flow, as you can see in the "before" vs. "after" [video](https://github.com/user-attachments/assets/c26af552-3f71-482f-a442-9b77701c9d03)

Reply:
@Parveshdhull yeah onboarding is different, that why we have 
```clojure
{:name      :screen/use-recovery-phrase-dark
     :metrics   {:track? true}
     :options   {:theme                  :dark
                 :modalPresentationStyle :fullScreen}
     :component enter-seed-phrase/view}
     ```
     which looks identical to your

Reply:
Removed new screen-id, Thank you very much for sharing this.